webrtc.html

```diff
@@ -517,6 +517,10 @@
 
         <ol>
           <li>
+            <p>Let <var>p</var> be a new promise</p>.
+          </li>
+
+          <li>
             <p>Append an object representing the current call being handled
```

Comment:
Where is obj.execute() defined?

Also, you say "execute the function from the front of the queue" below, which seems inconsistent. An object goes on the queue and a function comes off?